repository.rosinstall

```diff
@@ -0,0 +1,11 @@
+# .rosinstall file used by wstool.
+# Use this to download source-code dependencies.
+# This should ONLY be used if the dependency is not available on index.ros.org. 
+- git:
```

Comment:
Are you still using this? If so it should go in metapkg_deps rosinstall file, but i don't thing u need this anymore.